### DISCUSSION TITLE
"License: creativeml-openrail-m" has an icon that says "no selling images"?

### DISCUSSION THREAD

**Discussion by Jonseed** [civitai/civitai#168] [Q&A]
Why do the models like [Protogen Infinity](https://civitai.com/models/4398/protogen-infinity-official-release) have the [creativeml-openrail-m license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) with an icon that says "no selling images"? I don't think that is part of the license.

Section III.6 of the [license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) says:

> The Output You Generate. Except as set forth herein, Licensor claims no rights in the Output You generate using the Model. You are accountable for the Output you generate and its subsequent uses. No use of the output can contravene any provision as stated in the License.

There is nowhere else in the license that says you don't have rights to "sell" the output you generate.

If the restriction is based on [Dreamlike Photoreal v2.0](https://huggingface.co/dreamlike-art/dreamlike-photoreal-2.0) and its modified creativeml-openrail-m license, then that is inaccurate as well, as that license says:

> You are free to use the outputs (images) of the model for commercial purposes in teams of 10 or less

**Comment by JustMaier** [comment]
The permissions are set by model creators on a per model basis and may or may not reflect the licenses attached to their model. Instead, at this time, they are simply requests from the creator to people using their model. We had considered automatically picking an appropriate creative commons license based on their selection, but ultimately decided against it because of the uncertainty around licensing content created by AI.

In the cases of models that derive from Dreamlike, which has a modified license, the permissions are actually enforced by the Dreamlike license.

**Reply by green-anger** [reply]
> The permissions are set by model creators on a per model basis and may or may not reflect the licenses attached to their model.

How is it even possible? If you attach a license, the model must follow it. If you don't want to follow a particular license, you must not attach it. You cannot attach a license and change its rules. You can always create your own license, that's totally fine. There must be an option that says "custom license" and then you can add "no selling images" to it, but if you use, for example, CreativeML Open RAIL++-M, then it'll be contradictory. I completely agree with @Jonseed.
